frontend/app/templates/details.html

```diff
@@ -16,7 +16,7 @@
         <div class="col-md-6">
             <div class="school-details">
                 <h2>Mission</h2>
-                <div class="">{{ school.profile.mission_statement }}</div>
```

Comment:
you should be able to do this instead: <div class="">{{ school.profile.mission_statement | newline}}</div>
Doesn't really matter but I find this approach more readable.